shell/config/product/settings.js

```diff
@@ -102,13 +102,24 @@ export function init(store) {
     route:      { name: 'c-cluster-settings-links' }
   });
 
+  virtualType({
+    ifHaveType: FLEET.GIT_REPO,
+    labelKey:   'fleet.settings.label',
```

Comment:
This is more UX input, but I think `Fleet Settings` is better than just `Fleet`. I would double-check with UX

Reply:
that's good then. resolving this